snc-library.sh

```diff
@@ -141,62 +141,32 @@ function create_json_description {
             | ${JQ} ".clusterInfo.appsDomain = \"apps-${CRC_VM_NAME}.${BASE_DOMAIN}\"" >${INSTALL_DIR}/crc-bundle-info.json
 }
 
-function generate_pv() {
-  local pvdir="${1}"
-  local name="${2}"
-cat <<EOF
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: ${name}
-  labels:
-    volume: ${name}
-spec:
-  capacity:
-    storage: 100Gi
-  accessModes:
-    - ReadWriteOnce
-    - ReadWriteMany
-    - ReadOnlyMany
-  hostPath:
-    path: ${pvdir}
-  persistentVolumeReclaimPolicy: Recycle
-EOF
-}
-
-function setup_pv_dirs() {
+function setup_pv_dir() {
     local dir="${1}"
-    local count="${2}"
 
     ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo bash -x -s' <<EOF
-    for pvsubdir in \$(seq -f "pv%04g" 1 ${count}); do
-        mkdir -p "${dir}/\${pvsubdir}"
-    done
-    if ! chcon -R -t svirt_sandbox_file_t "${dir}" &> /dev/null; then
+    mkdir -p "${dir}"
+    if ! chcon -R -t container_file_t "${dir}" &> /dev/null; then
```

Comment:
https://github.com/kubevirt/hostpath-provisioner#deployment-in-openshift it says we don't need to relabel the directory and driver suppose to take care of it, do test it and if that is the case then we just need to create the directory not to change the label.